sound_play/scripts/is_speaking.py

```diff
@@ -46,10 +46,13 @@ def callback(self, msg):
 
     def speech_timer_cb(self, timer):
         self.pub_speech_flag.publish(
-            std_msgs.msg.Bool(self.is_speaking))
+            Bool(data=self.is_speaking))
 
 
 if __name__ == '__main__':
-    rospy.init_node('is_speaking')
-    app = IsSpeaking()
-    rospy.spin()
+    rclpy.init()
+    is_speaking_node = IsSpeakingNode()
+    rclpy.spin(is_speaking_node)
+    is_speaking_node.destroy_node()
+    del is_speaking_node
```

Comment:
Assume that this was necesary to avoid any error, did you find this solution somewhere else? If so please answer in this thread with the link just to proper document this.

Reply:
That part of the solution was made on the previous [port](https://github.com/ros-drivers/audio_common/commit/3b86f459680db2ba12fd71859db8f00d332445b1#diff-059701fa3e2765f3c4b0a0c01536081f3f6d9b181e10177281224df19757459f) (take a look at the last part of the file).